README.md

```diff
@@ -2,6 +2,73 @@
 
 This repository contains attestation services assembled using Veraison components.
 
+## Getting Started
+
+This section contains the instructions for creating a test deployment of
+Veraison services and trying out the end-to-end attestation flow using sample
+inputs.
+
+### Requirements
+
+This should work on most Linux systems. You need to perform the following
+setup:
+
+- Install git
+- Install Docker, and make sure that the current user is in the `docker` group.
+- Install jq
+- (optionally) Install tmux -- this only needed if you want to use it.
+
+On Ubuntu you can do this with:
+
+    sudo apt install git docker.io jq tmux
+    sudo usermod -a -G docker $USER
+    newgrp docker
+
+### Creating a deployment
+
+You can build, deploy, and start Veraison services with the following sequence
+of commands:
+
+    git clone https://github.com/veraison/services.git
+    cd services
+    make docker-deploy
+
+The whole process might take a few minutes. Once the above finishes, Veraison
+services should be running inside Docker containers. You can use the deployment
+frontend script to check their status (you can set it up by sourcing the
+deployment's `env.bash`):
+
+    source deployments/docker/env.bash
+    veraison status
+
+See the output of `veraison -h` for the full list of available commands
+
+### End-to-end flow
+
+The end-to-end flow can be performed with the `end-to-end/end-to-end` script.
+
+Before evidence can be attested, trust anchors and reference values need to
+provisioned. These are contained within
```

Comment:
The flow nicely explains the default (psa) flow.

If some one wants to run CCA flow, can you please add a line to clarify how and what small change the use has to do to switch from default psa case ?

Reply:
That's a good point, however this does not belong here. I'll add  a note about switching the scheme to the explanation of the end-to-end flow inside `end-to-end/README.md`.